### DISCUSSION TITLE
[SC]: tabella comuni

### DISCUSSION THREAD

**Discussion by Hdemografici** [italia/ansc#877] [Q&A] [answered]
### Ambiente

pre-produzione

### Servizio

R009 - Servizi cooperativi validazione evento

### idOperazione

assente

### Provenienza segnalazione

Software House

### Comune di riferimento e operatore

assente

### Descrivi il problema

Buongiono controllando la tabella dei comuni ho trovato divesi comuni che hanno la stessa denominazione, provincia e codice istat 
e data cessazione e costituzione conseguenti
ad esempio
ABBASANTA 92001 CA
che risulta istituito 02/11/1936 cessato il 23/08/1946
poi istituito 24/08/1946 cessato 15/06/1950
nuovamente istituito 16/06/1950 cessato 19/08/1974
quando poi c'è stato il cambio provincia da CA a OR

come questo ce ne sono diversi, controllando le tabelle di ISTAT non ho trovato riscontri quindi non so è errata la tabella oppure mi sfugge qualcosa.

**Comment by mttfranci** [comment]
Salve @Hdemografici , solo per conferma, stiamo parlando della tabella 3, 'ANPR Archivio comuni - csv' del portale ANPR?

https://www.anagrafenazionale.interno.it/area-tecnica/tabelle-di-decodifica/
In particolare? 
https://www.anagrafenazionale.interno.it/wp-content/uploads/ANPR_archivio_comuni.csv

Saluti

Assistenza ANSC

**Comment by Hdemografici** [comment]
esattamente quella, gli id comune dell'esempio precedente sono 9, 17678, 17679
per il primo avevo indicato 1936 in realtà è 1934

**Comment by migmar68** [comment]
Tutti i casi del genere che ho notato io, corrispondevano sempre a eventi di variazione territoriale: ad esempio un pezzo del Comune che veniva ceduto, oppure assorbito.
Nel caso di Abbasanta del 1946, dovrebbe corrispondere alla creazione del Comune di Norbello.

**Comment by mttfranci** [comment, marked as answer]
@Hdemografici l'elenco è manutenuto nell'ambito del progetto dell' anagrafe <https://github.com/italia/anpr>. 
Come indicato anche da @migmar68, in generale, ogni variazione è derivante da qualche variazione avvenuta sul comune.
